module/x/gravity/types/key.go

```diff
@@ -243,11 +278,25 @@ func GetOutgoingTxPoolKey(fee InternalERC20Token, id uint64) string {
 	return ConvertByteArrToString(r)
 }
 
+// GetOutgoingTxBatchNoncePrefix returns the following format
+// prefix     nonce
+// [0xa][0 0 0 0 0 0 0 1]
+func GetOutgoingTxBatchNoncePrefix(nonce uint64) string {
+	return OutgoingTXBatchKey + ConvertByteArrToString(UInt64Bytes(nonce))
+}
+
 // GetOutgoingTxBatchKey returns the following key format
 // prefix     nonce                     eth-contract-address
 // [0xa][0 0 0 0 0 0 0 1][0xc783df8a850f42e7F7e57013759C285caa701eB6]
 func GetOutgoingTxBatchKey(tokenContract EthAddress, nonce uint64) string {
-	return OutgoingTXBatchKey + tokenContract.GetAddress() + string(UInt64Bytes(nonce))
```

Comment:
My changes changing order to match comment 
// [0xa][0 0 0 0 0 0 0 1][0xc783df8a850f42e7F7e57013759C285caa701eB6] wrong order or comment?

was
`OutgoingTXBatchKey + tokenContract.GetAddress() + string(UInt64Bytes(nonce))`
now
`GetOutgoingTxBatchNoncePrefix(nonce) + tokenContract.GetAddress()`